src/main/resources/accessions.vm

```diff
@@ -4,8 +4,8 @@
       <link rel="stylesheet" type="text/css" href="/style.css">
   </head>
   <body>
-  <h1>Accessions</h1>
-    <table>
+  <h1 style="padding: 10px;">Accessions</h1>
```

Comment:
In general, it's best to put all the style information in a css class to fully separate page presentation from structure.  If this style shouldn't apply to all h1 elements, give it a class or id and apply the style selectively in the CSS.